### PR TITLE
Style polish

### LINKED ISSUE #18: Style polish for 0.0.1
- [x] Border radius should be 1.0
- [x] Page selector spacing between buttons with columns
- [x] Disabled state styles (black text, no border)
- [x] Active state styles (black border)
- [x] All non-testing page buttons that are currently in the `hidden` vec should be displayed by default, but disabled

### PR DESCRIPTION
**Pull request by kn0wmad**
WIP

Closes #18

### CODE REVIEW

File: src/page.rs [hunk 1]
```diff
@@ -6,7 +6,7 @@ pub mod site;
 pub mod testing;
 pub mod view;
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, Copy, PartialEq)]
```
Comment (cryptoquick) [inline src/page.rs]: Do you really need to derive Copy? Usually Clone does the trick.

File: src/ui/page_selector.rs [hunk 2]
```diff
@@ -29,50 +30,55 @@ impl PageSelector {
...
         PageSelector {
             buttons,
             active_page: PageType::Publish,
+            disabled_pages: vec![
+                PageType::Dashboard,
+                PageType::Feed,
+                PageType::Settings,
+                PageType::Site,
+                PageType::Testing,
+            ],
```
Comment (cryptoquick) [inline src/ui/page_selector.rs]: I love this approach. That said, do you think we should still have the Testing page? We could probably just get rid of it.

File: src/ui/page_selector.rs [hunk 3]
```diff
@@ -83,6 +89,9 @@ impl PageSelector {
                         &mut page_button.button_state,
                         Text::new(page_button.label_text.clone()).size(16),
                     )
+                    .style(style::Button::Disabled {
+                        disabled: vec![page_button.page_type] == *disabled_pages,
```
Comment (cryptoquick) [inline src/ui/page_selector.rs]: Why are you comparing the two vecs? Shouldn't you be checking to see if it's in the vec? Hm... Perhaps an enum with a match statement will be more idiomatic... We'll have to think of that on a call.

**Review by cryptoquick — CHANGES_REQUESTED** [review]

File: src/ui/page_selector.rs (hunk 2 above)
Comment (kn0wmad) [reply src/ui/page_selector.rs]: Agreed, it can go.

File: src/ui/page_selector.rs (hunk 3 above)
Comment (kn0wmad) [reply src/ui/page_selector.rs]: Yeah, let's do that

**Review by kn0wmad — COMMENTED** [review]
Addressed

**Review by cryptoquick — APPROVED** [review]

**Pull request merged**